app/index.js

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+const port = process.env.PORT || 3000;
+const app = express();
+
+app.use(function (req, res, next) {
+  res.setHeader("Access-Control-Allow-Origin", ["*"]);
```

Comment:
From what I'm aware, just `'*'` is enough. You don't need to write it in an array.